crates/edr_solidity/src/source_map.rs

```diff
@@ -159,7 +160,16 @@ pub fn decode_instructions(
         let source_map = &source_maps[instructions.len()];
 
         let pc = bytes_index;
-        let opcode = OpCode::new(bytecode[pc]).expect("Invalid opcode");
+        let opcode = if let Some(opcode) = OpCode::new(bytecode[pc]) {
+            opcode
+        } else {
+            debug!("Invalid opcode {} at pc: {}", bytecode[pc], pc);
```

Comment:
nit: I prefer to not to import these macros from log and just write `log::debug!` so that it's easy to differentiate from `tracing::debug!`

Reply:
+1